UnityProject/Assets/Scripts/Health/Living/HumanHealth/PlayerHealth.cs

```diff
@@ -76,6 +76,8 @@ protected override void OnDeathActions()
 			if (isServer)
 			{
 				EffectsFactory.BloodSplat(transform.position, BloodSplatSize.large, bloodColor);
+				string descriptor = player.Script.characterSettings.PossessivePronoun();
+				Chat.AddLocalMsgToChat($"<b>{playerName}</b> seizes up and falls limp, {descriptor} eyes dead and lifeless...", (Vector3)TransformState.HiddenPos);
```

Comment:
When you add a local message you need to tell the chat system where it's happening at, which is what that last parameter is for. You should pass registerPlayer.WorldPositionServer here which represents the position the server thinks this player is at, instead of HiddenPos. The message wasn't showing up for me when testing with killing dummy player. HiddenPos is a special position used for things that aren't supposed to be visible to the players, which shouldn't be passed to this method.

Reply:
oh, alrighty, thanks. I'll patch it up once I get a minute!